group_vars/all

```diff
@@ -13,6 +13,7 @@ admin_server: w2k8-1.ad.sec.cloudera.com
 krb5_kdc_type: mit
 
 cdh_version: 5.8.3
+cdh_package_version: 5.8.3-1.cdh5.8.3.p0.2
```

Comment:
Should we define only patch version here, and make `cdh_package_version` by merging cdh_version and the patch version? I thought about this option, but it may be broken in the future if Cloudera provides package like 5.8.3-0.cdh5.8.3.p0.2 (not `-1`).  Thus, this change is okay.

Reply:
When we change the version in the future, we have a review phase. If cdh_version and cdh_package_version is not consistent, we'll find the inconsistency at the review.